<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mupdate</string>
  </property>
  <widget class="QListWidget" name="list_serverListing">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>110</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_retrieve">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Retrieve Listing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_download">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Download Selected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_change">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_downloadPath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_downloadPath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Download Path</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_serverListing">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server Listing</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_fileProgress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;No files selected&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>x64.png</normaloff>x64.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
